edge/services/helloSecretWorld/CreateService.md

```diff
@@ -0,0 +1,262 @@
+# Creating Your Own Hello Secret World Edge Service
+
+Follow the steps in this page to create your first Horizon edge service that uses the secrets manager.
+
+## Preconditions for Developing Your Own Service
+
+1. If you have not already done so, complete the steps in these sections:
+
+  - [Preconditions for Using the Hello Secret World Example Edge Service](README.md#preconditions)
+  - [Using the Hello Secret World Example Edge Service with Deployment Pattern](README.md#using-hello-secret-pattern)
+
+2. If you are using macOS as your development host, configure Docker to store credentials in `~/.docker`:
+
+  - Open the Docker **Preferences** dialog
+  - Uncheck **Securely store Docker logins in macOS keychain**
+
+3. If you do not already have a docker ID, obtain one at https://hub.docker.com/ . Log in to Docker Hub using your Docker Hub ID:
+
+  ```bash
+  export DOCKER_HUB_ID="<dockerhubid>"
+  echo "<dockerhubpassword>" | docker login -u $DOCKER_HUB_ID --password-stdin
+  ```
+
+  Output example:
+
+  ```bash
+  WARNING! Your password will be stored unencrypted in /home/pi/.docker/config.json.
+  Configure a credential helper to remove this warning. See
+  https://docs.docker.com/engine/reference/commandline/login/#credentials-store
+
+  Login Succeeded
+  ```
+
+4. Create a cryptographic signing key pair. This enables you to sign services when publishing them to the exchange. **This step only needs to be done once.**
+
+  ```bash
+  hzn key create "<x509-org>" "<x509-cn>"
+  ```
+
+  where `<x509-org>` is your company name, and `<x509-cn>` is typically set to your email address.
+
+5. Install `git`, `jq`, and `make`:
+
+  On **Linux**:
+
+  ```bash
+  sudo apt install -y git jq make
+  ```
+
+  On **macOS**:
+
+  ```bash
+  brew install git jq make
+  ```
+
+## <a id=build-publish-your-hw></a> Building and Publishing Your Own Hello Secret World Example Edge Service
+
+1. Clone this git repo:
+
+  ```bash
+  cd ~   # or wherever you want
+  git clone git@github.com:open-horizon/examples.git
+  ```
+2. Checkout the branch that corresponds to your horizon CLI version. To get the branch name, remove the last bullet and any numbers after it, then prepend a `v` at the beginning:
```

Comment:
This whole step bothers me and I think it's old. I guess we need to ask Troy if it's really needed.